prettier/prettier-shim

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Modify any relative path arguments to be absolute
+args=()
+
+for arg; do
+  abs=$PWD/$arg
+
+  if [[ -e "$abs" ]]; then
+    args+=("$abs")
+  else
+    args+=("$arg")
+  fi
+done
```

Comment:
This seems _slightly_ shaky, because on the off-chance that `arg` is an absolute path (`/foo.txt`) that also exists within the currently directory (`$PWD/foo.txt`), `abs` will be `$PWD//foo.txt` and be picked instead of the intended file.

Reply:
I can at least say that when Restyler invokes these images, all paths are [passed as relative](https://github.com/restyled-io/restyler/blob/95769cf68fa6a0e66e08576c1d6dc3504c9648f0/src/Restyler/Restyler/Run.hs#L423-L425).

Not that we shouldn't code against that changing though.

Reply:
Works for me.

Reply:
Pushed my Bash understanding past its limits. https://github.com/restyled-io/restylers/pull/1108/files#diff-502bc537b5e00f9f2cfde0f849a39c67b7c1b710893b39e0681aa9d5d7c1797eR51-R53

(`''\` there is a Nix escape sequence to escape `${`.)

Reply:
Ohhhh I see, you're just checking *every* argument, without knowledge of whether it's intended to be a path or not. Interesting.